torch/csrc/distributed/rpc/rref_context.cpp

```diff
@@ -160,9 +166,15 @@ void RRefContext::delUser(
   {
     std::lock_guard<std::mutex> lock(destroyedMutex_);
     if (!destroyed_) {
-      auto fm = agent_->send(
+      rpc::RpcRetryOptions options;
+      options.maxRetries = kNumRrefControlMsgRetries;
+      // Sending an RRefUserDelete causes the receiver to run delForkOfOwner,
+      // which is now idempotent. See the comment at RRefContext::delForkOfOwner
+      // for more details.
+      auto fm = agent_->sendWithRetries(
           agent_->getWorkerInfo(owner),
-          RRefUserDelete(rrefId, forkId).toMessage());
+          RRefUserDelete(rrefId, forkId).toMessage(),
+          options);
```

Comment:
looks like the `options` code occurs in multiple places, does it make sense if we assign a default value to the `options` arg in `sendWithRetries` and remove the duplications in the call sites?

Reply:
Sounds good - we already have the default options set in RpcRetryOptions struct - we can just fall back on those.